artiq/dashboard/moninj.py

```diff
@@ -660,6 +660,8 @@ async def mi_connector(self):
                 await asyncio.sleep(10.)
                 self.reconnect_mi.set()
             else:
+                logger.info("ARTIQ dashboard %s connected to %s",
```

Comment:
Printing the version again doesn't make a lot of sense.
There could be a separate message for the version then.

Reply:
Can I add more info into the log message for both master and proxy?
ie.: `...connected to ::1 master`
`...connected to  ::1 moninj proxy`

Reply:
Sure. Just have consistent and good messages in both cases.